server/src/schema.js

```diff
@@ -324,9 +325,16 @@ export const Schema = [
     sendTestPush(vars: SendTestPushInput): Boolean  # send a push notification to the requesting device
   }
 
+  type Subscription {
+    message(eventId: Int,groupId: Int): Message
+    eventResponse(eventId: Int!): EventResponse
```

Comment:
shouldn't we just send down updates to this as part of the event? sure it's a little less efficient but would save some extra code. Any time any part of an event changes (including the responses) just send the new event down.

Reply:
It's many times less efficient not just a little

Reply:
went with this in the end just to speed up dev work